TestLogin/TestLogin/TestLogin/MainPage.xaml.cs

```diff
@@ -13,6 +13,22 @@ public partial class MainPage : ContentPage
         public MainPage()
         {
             InitializeComponent();
+            loginButton.Clicked += LoginButton_Clicked;
+            var username = new Entry {Text = "lahiru"};
```

Comment:
Use correct data type here instead of var

Reply:
Ok mr.dilan. I will fix that and send another update